scripts/quick-helm-install.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+SMALL_INSTALL=false
+
+case "${1:-}" in
+    -h|--help)
+        echo -e "Usage:\n\tquick-helm-install.sh [options]"
+        echo " "
+        echo "Installs StackRox via Helm charts, requires Helm."
+        echo " "
+        echo "options:"
+        echo "-h, --help                show brief help"
+        echo "-s, --small               reduce StackRox resource requirements for small clusters"
+        exit 0
+        ;;
+    -s|--small)
+        SMALL_INSTALL=true
+        ;;
+esac
+
+echo "Adding the stackrox/helm-charts/opensource repository to Helm."
+
+helm repo add stackrox https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource/
+
+echo "Generating STACKROX_ADMIN_PASSWORD"
+
+STACKROX_ADMIN_PASSWORD="$(openssl rand -base64 20 | tr -d '/=+')"
+
+echo "Installing stackrox-central-services"
+
+helm install -n stackrox --create-namespace stackrox-central-services stackrox/stackrox-central-services --set central.adminPassword.value="${STACKROX_ADMIN_PASSWORD}"
+
+if $SMALL_INSTALL; then
```

Comment:
Let's be explicit here 

```suggestion
if [[ "$SMALL_INSTALL" == "true" ]]; then
```

Reply:
I always considered it bad form to compare a boolean with true in an if statement rather than evaluating it directly. Why is it better to do this in bash?

Reply:
You ask valid question: "why", but the answer is not so trivial. Short answer from me: because it is safer to do so.

For the full answer I would need you redirect to a book, because I cannot find a quick justification in stackoverflow or other sources (they also suggest it, but they not fully explain the "why" part)